### ISSUE TITLE
[Functional - Bot Framework Web Chat- Enable Emoji]: After typing keyword ':)', it is not converted to emoji.

### ISSUE THREAD

**Issue by v-duku** [microsoft/BotFramework-WebChat#4016]
### User Experience:
If After typing keyword ':)', it is not converted to emoji, then user will be disoriented and will not be able to use this functionality. The user will also be facing issues while accessing the emojis.

**Note**: Login Credentials must not be provided within the bug.

### Test Environment: 
**OS Version:** 21H1 
**OS Build:** 19043.1110
**Browser:** Microsoft Edge Version 94.0.972.2 (Official build) dev (64-bit)

### Repro steps:

1. Open URL: Web Chat: Full-featured bundle (microsoft.github.io)
2. Web chat bundle will open.
3. Navigate on the web chat bot and enter below keywords in chatbot.
4. Type :) and note that it is converted to emoji
5. Type Ctrl + Z and note that the emoji is reverted to the original emoticon.
6. Cards related to the keyword will appear, navigate on the cards.
7. Verify the issue.

### Actual Result
After typing keyword ':)', it is not converted to emoji.

### Expected Result
After typing keyword ':)', it should be converted to emoji.

**Note**: This issue exists with similar controls throughout the application.

### Reference Links
[Accessibility Insights](https://accessibilityinsights.io/)! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**Comment by v-duku** [comment]
#Functional;#A11ySev1;#HCL;#NonBenchmark;#Win10-Edge(Chromium);#E2E-HCL-BotFrameworkWebChat-Aug21;

**Comment by compulim** [comment]
@v-duku could you try this out independently with Adaptive Cards (via preview mode of [AC designer](https://adaptivecards.io/designer/))?

Our `help` command is sending multiple AC, and the root cause of the issue could be on AC side instead.

**Comment by compulim** [comment]
In your step 4, Web Chat converted `:)` to emoji.

But in your step 5, <kbd>CTRL</kbd>+<kbd>Z</kbd> actually undo-ed the emoji conversion.

Because of the undo keystroke, we are sending `:)` in plain text, but not Emoji.

**Comment by compulim** [comment]
Recording showing `:)` will convert to 😊. And <kbd>CTRL</kbd> + <kbd>Z</kbd> will undo it.

![recording (24)](https://user-images.githubusercontent.com/1622400/130521700-7e4b3895-a2c1-4eb2-935d-c47e8104ceed.gif)

**Comment by v-shikch** [comment]
As verified issue on URL: Web Chat: Full-featured bundle (microsoft.github.io) on Browser: Edge Chromium Version 96.0.1028.0 (Official build) dev (64-bit), here issue is fixed. while typing the value as ":)" it's converted into emoji and after using 'ctrl+z" it's again converting in original text. Hence closing the bug.

**Comment by compulim** [comment]
Thanks @v-shikch and @v-duku. Appreciated.
